CHANGELOG.md

```diff
@@ -30,55 +39,67 @@ All notable changes to this project will be documented in this file.
 * Kotlin metadata to 0.2.0
 
 ## 2.1.0
+
 * Fixed issue that was preventing the plugin from been applied in the `plugins` block
 * Converted `build.gradle`s to `build.gradle.kts`
 
 ## 2.0.0
+
 * Upgrade kotlin to 1.5.31 in JsPlugin
 * MockingBird plugin
 
 ## 1.15.0
+
 * Support for iosSimulatorArm64
 * HMPP support
 
 ## 1.14.0
+
 * Added support to capture objects and lists while running a test with LOCAL_THREAD mode
 * **[DEPRECATED]** `Slot()` use `slot()` instead
 * **[DEPRECATED]** `CapturedList()` use `capturedList()` instead
 
 ## 1.13.0
+
 * **breaking** If there are multiple mocked responses matching a mock invocation, the one that was added last will be used
-* Introduced testing mode, before the behavior was always MULTI_THREAD now it is possible to set LOCAL_THREAD mode to avoid argument freeze on mock invocation
+* Introduced testing mode, before the behavior was always MULTI_THREAD now it is possible to set LOCAL_THREAD mode to avoid
+  argument freeze on mock invocation
 
 ## 1.12.0
+
```

Comment:
Are newlines after release headers added by Merlin automatically?

Reply:
nope it was my format on the file itself, merlin just replaces the #master with the new release tag